src/graphql/mod.rs

```diff
@@ -100,11 +100,11 @@ impl QueryClient {
         let body = Q::build_query(variables);
 
         debug!("url: {:?}", &self.api_url);
-        debug!("GraphQL query: \n{}", body.query);
+        debug!("query: \n{}", body.query);
 
         let response: Result<Response<Q::ResponseData>, APIError> =
             self.retry_request::<Q>(body, handler).await;
-        debug!("GraphQL response: {:#?}", response);
+        debug!("response: {:#?}", response);
```

Comment:
I think we don't need a debug log here for `response` anymore, we already cover all of them inside `retry_request`